packages/cli-lib/lib/handleFieldsJs.js

```diff
@@ -0,0 +1,113 @@
+const fs = require('fs');
+const path = require('path');
+const { logger } = require('../logger');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { getExt } = require('../path');
+const i18nKey = 'cli.commands.upload';
+const { getCwd } = require('@hubspot/cli-lib/path');
+const fileResolver = {
+  apply: (target, thisArg, args) => {
+    let filePath = args.shift();
+
+    // Get the path to the folder containing the fields.js file through a stack trace so we can resolve relative paths.
+    const line = new Error().stack.split('\n')[2];
+    const callerPath = line.slice(
+      line.lastIndexOf('(') + 1,
+      line.lastIndexOf('/') + 1
+    );
+    const absoluteSrcPath = path.resolve(getCwd(), callerPath, filePath);
+    return target(absoluteSrcPath, ...args);
+  },
+};
```

Comment:
I do not like this method of the stack + proxy to resolve relative paths since it feels fragile, but I'm not sure of a better way to do this.

Maybe this isn't the place to have the loadPartials/loadJson methods in the first place, and both should be moved to a (future) library that includes some other utilities for writing fields.js files (such as Field objects).

Reply:
After some IRL conversation with @miketalley, I've decided that `loadJSON` and `loadPartial` don't really fit in this PR & have removed them.